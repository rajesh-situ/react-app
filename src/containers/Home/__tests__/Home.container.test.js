import React from 'react';
import renderer from 'react-test-renderer';
import HomeContainer from '../Home.container';

describe('the home container', () => {
  it('should render correctly', () => {
    const component = renderer.create(<HomeContainer />).toJSON();
    expect(component).toMatchSnapshot();
  });
});

