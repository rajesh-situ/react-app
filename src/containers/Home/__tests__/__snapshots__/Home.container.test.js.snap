// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the home container should render correctly 1`] = `
<div>
  Hey there!
</div>
`;
